website/docs/components/rich-tooltip/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,215 @@
+
+#### Size
+
+![Sizes of the InfoText](/assets/components/rich-tooltip/rich-tooltip-info-text-sizes.png)
+
+#### Icon
+
+![Sizes of the InfoText](/assets/components/rich-tooltip/rich-tooltip-info-text-icon.png)
+
+#### Underline
+
+Only available in the InfoText component in Figma, the `isUnderlineOnly` property allows the component to be used **inline** a block of text like a paragraph.
+
+![Sizes of the InfoText](/assets/components/rich-tooltip/rich-tooltip-info-text-underline.png)
+
+#### Variants
+
+- `size`: `small`, `medium`, `large`
+- `hasText`: boolean; defaults to true
+- `iconPosition`: `leading`, `trailing` (default), `only`, `none`
+
+### Custom toggle
+
+While almost any element can be used to toggle the Rich Tooltip, custom elements must conform to the following usability and accessibility success criteria:
+
+- Must have a minimum target area of 24x24 pixels.
+- Must have a minimum contrast ratio between the background/surface color of 3:1.
+- Should use actionable language or be paired with a label to communicate that additional context is obfuscated away from the user.
+
+More information about this success criteria can be found in the [accessibility](?tab=accessibility) section.
+
+!!! Dont
+
+Don’t use a [Badge](/components/badge) or [BadgeCount](/components/badge-count) as the toggle for the Rich Tooltip; Badges are intentionally non-interactive elements and don't visually communicate that content is hidden behind a `hover`, `click`, or `focus` interaction.
+
+![Example of using the Rich Tooltip with a badge](/assets/components/rich-tooltip/rich-tooltip-trigger-badge-dont.png)
+!!!
+
+!!! Do
+
+Instead, place the InfoText component inline with the Badge to communicate the existence of a Rich Tooltip.
+
+![Example of pairing the default toggle with a Badge](/assets/components/rich-tooltip/rich-tooltip-trigger-badge-do.png)
+!!!
+
+!!! Dont
+
+Don’t toggle a Rich Tooltip from a form element like a [Text Input](/components/form/text-input), [Select](/components/form/select), or [Textarea](/components/form/textarea).
+
+![Example of toggling a Rich Tooltip from a form element](/assets/components/rich-tooltip/rich-tooltip-trigger-form-input-dont.png)
+!!!
+
+!!! Do
+
+Instead, use the [Label](/components/form/primitives#formlabel) and [HelperText](/components/form/primitives#helpertext) primitives to communicate details about the field. If _absolutely_ necessary to provide more details about a form element, use the InfoText inline within [Helper Text](/components/form/primitives#formhelpertext).
+
+![Example within Helper Text](/assets/components/rich-tooltip/rich-tooltip-trigger-helper-text-do.png)
+
+However, interactive elements in HelperText will not be read out as interactive to users with screen readers, only the text will be read. More details on this recommendation can be found in the [Helper Text documentation](/components/form/primitives?tab=code#formhelpertext-1).
+!!!
```

Comment:
Should we recommend this if it isn't accessible? What about putting the InfoText icon next to the field (like in the badge example above)? Or recommend using a reveal for the additional details instead?

Reply:
This was an attempt to mimic the content we have [here](https://helios.hashicorp.design/components/form/primitives?tab=code#formhelpertext-1) about links within Helper Text.

I'm torn on this:
- In an ideal world all of the necessary context would be expressed through the label and the helper text, though sometimes we've seen this isn't true
- It is a beneficial use case if more information is provided, but the fact it maybe not be apparent to screen readers is 👎 
- Does the idea that we've termed this as "non-essential" information have any impact here?

Reply:
The difference with the link you provided is that it's not provided as a recommendation; it's provided as a possibility with a warning. I understand that there may be a need to provide additional context, which is why I provided a couple of alternative methods instead of including this in the HelperText. Why not recommend either of those options instead, as they're bound to lead to a more accessible experience? The "reveal" pattern is already used quite frequently in HCP forms.

Reply:
Ah, ok I understand now, agreed. I've updated this to recommend against and use a reveal instead.